README.md

```diff
@@ -71,7 +71,6 @@ See the examples in `tests/horn/{pos, neg}` eg
 
 - [sum](tests/horn/pos/ple_sum.smt2)
 - [list00](tests/horn/pos/ple_list00.smt2)
-- [list03](tests/horn/pos/ple_list03.smt2)
```

Comment:
This could be `tests/horn/neg/ple_list03.smt2` instead.

Reply:
Good call! I've added this link back in, adjusted to point to `neg` rather than `pos`.